index.js

```diff
@@ -265,7 +265,21 @@ const getCountryNeighbors = (country) => {
 
   return data.filter(({ neighbors }) => neighbors.includes(foundCountry.iso.alpha_2));
 };
-
+/**
+ * Returns an array of objects containing all countries 
+ * which areas satisfies areaLower<=Country Area<=areaUpper
+ * @param { Number } areaLower  Minimum area
+ * @param { Number } areaUpper  Maximum area
+ * @param {sys} Area system kilometers^2 = km, miles^2 = mi
+ * @returns {Country[]} An array of country objects
+*/
+const getCountriesByRangeSize =(areaLower,areaUpper,sys="km")=>{
+  if(sys!="sys"&&sys!="mi"){sys="km";}
+  let countries = data.filter(country=>{
+    return country.area[sys+"2"]>=areaLower && country.area[sys+"2"]<=areaUpper;
```

Comment:
What if `sys` is passed as "sys", how this code will work?

Reply:
Sorry, my bad. I should have put "km" instead of sys. It was a kind of validation to ensure the sys parameter.I'll fix it now

Reply:
I already submit the solution to this bug and its tests